bundles/io/org.eclipse.smarthome.io.rest.test/src/test/groovy/org/eclipse/smarthome/io/rest/test/filter/CorsFilterTest.groovy

```diff
@@ -34,6 +34,8 @@ class CorsFilterTest {
     private static final String HTTP_GET_METHOD = "GET"
     private static final String HTTP_OPTIONS_METHOD = "OPTIONS"
 
+    private static final String CONTENT_TYPE_HEADER = "content-type";
```

Comment:
> I don't think we should have this method. Header names are case-insensitive as described by the specifications 

Would this not be the argument to have a method such as `containsHeaderWithCaseInsensitiveValue`?  In fact all header names should be compared insensitive to case.  Perhaps I don't understand your comment?